src/faldbt/project.py

```diff
@@ -173,21 +186,26 @@ def __hash__(self) -> int:
     def get_depends_on_nodes(self) -> List[str]:
         return self.node.depends_on_nodes
 
-    def _get_hook_paths(self, hook_type: str, keyword: str = "fal") -> List[str]:
+    def _get_hooks(self, hook_type: str, keyword: str = "fal") -> List["Hook"]:
```

Comment:
ATL: if it is imported, why not `List[Hook]`?

Reply:
It is not imported, the import for this is guarded by `TYPE_CHECKING` (to prevent circular dependencies) so we can't directly reference the name but rather use the stringified version.